packages/jest-snapshot/src/index.js

```diff
@@ -48,6 +48,27 @@ const cleanup = (hasteFS: HasteFS, update: SnapshotUpdateState) => {
   };
 };
 
+function isObject(item) {
+  return (item && typeof item === 'object' && !Array.isArray(item));
+}
+
+function deepMerge(target, source) {
+  let mergedOutput = Object.assign({}, target);
+  if (isObject(target) && isObject(source)) {
+    Object.keys(source).forEach(key => {
+      if (isObject(source[key]) && !source[key].$$typeof) {
+        if (!(key in target))
+          Object.assign(output, {[key]: source[key]});
```

Comment:
I'm not sure where `output` is defined? Should this not be:

```js
Object.assign(mergedOutput, {[key]: source[key]});
```

Reply:
Thanks for catching that, it was late when I submitted this and I overlooked it.